### PR TITLE
docs: update roadmap: data market -> data exchange

### PR DESCRIPTION
**Pull request by youngjoon-lee**
pls review carefully since i wrote this PR on mobile. sorry about that

### CODE REVIEW

**Review by hansol-medi — APPROVED** [review]
All the `market` is changed to `exchange`. If anyone have better words or expressions, leave comments please.

File: .gitbook/overview/roadmap.md [hunk 1]
```diff
@@ -22,22 +22,22 @@
...
 - Improve governance process
 
-### 2022 Q3: Data Market Protocol v0 on Testnet
+### 2022 Q3: Data Exchange Protocol v0 on Testnet
 
-- Evaluate data market protocol v0 on Panacea Testnet
+- Evaluate data exchange protocol v0 on Panacea Testnet
   - Technical evaluation
   - Product-wise evaluation
 - Revise whitepaper
```
Comment (tjyoon0324) [inline .gitbook/overview/roadmap.md:30]:
```suggestion
- Update and release new version of whitepaper
```

File: .gitbook/overview/roadmap.md [hunk 2]
```diff
@@ -46,19 +46,19 @@
...
   - Cross-chain NFT (compatible with NFT marketplaces)
 
-### 2023 Q3: Healthcare Services on Data Market Protocol
+### 2023 Q3: Healthcare Services on Data Exchange Protocol
 
-- Release data market protocol v1 on Panacea mainnet
-- Integrate MediBloc healthcare services with data market
+- Release data exchange protocol v1 on Panacea mainnet
+- Integrate MediBloc healthcare services with data exchange protocol
 - Revise whitepaper
```
Comment (tjyoon0324) [inline .gitbook/overview/roadmap.md:53]:
```suggestion
- Update and release new version of whitepaper
```

**Review by tjyoon0324 — APPROVED** [review]

File: .gitbook/overview/roadmap.md (hunk 2 above)
Comment (tjyoon0324) [reply .gitbook/overview/roadmap.md:53]: Thanks Youngjoon! The change looks great and I just added a slight suggestion to change our wording regarding the whitepaper revision.

**Review by gyuguen — APPROVED** [review]
From what I can see, everything seems to have been fixed normally.

File: .gitbook/overview/roadmap.md (hunk 1 above)
Comment (youngjoon-lee) [reply .gitbook/overview/roadmap.md]: thank you.
done: https://github.com/medibloc/panacea-core/commit/1c13ab2c997ee2016c7be4a194006feef9f8be98

File: .gitbook/overview/roadmap.md (hunk 2 above)
Comment (youngjoon-lee) [reply .gitbook/overview/roadmap.md]: done https://github.com/medibloc/panacea-core/commit/a1d623682500a823ff80808dbb4b0f9e89786bf0

**Pull request merged**